examples/resources/port_webhook/main.tf

```diff
@@ -0,0 +1,20 @@
+resource "port_webhook" "github" {
+  identifier = "github"
+  title      = "Github"
+  icon       = "Terraform"
```

Comment:
Why not icon github?

Reply:
That depends, but I think we want to see in the ingest modal an icon of terraform and not Github ?